gwconfig/config.go

```diff
@@ -68,6 +68,8 @@ type (
 		WorkerID                       string                 `json:"workerId"`
 		WorkerType                     string                 `json:"workerType"`
 		WorkerTypeMetadata             map[string]interface{} `json:"workerTypeMetadata"`
+		WstAudience                    string                 `json:"wstAudience"`
+		WstServerURL                   string                 `json:"wstServerURL"`
```

Comment:
Please update the usage docs (`generic-worker --help`) with details about config settings `wstAudience` and `wstServerURL`. Note that required properties and optional properties are listed in separate sections of the usage docs.

nit: the Go naming convention prefers uppercase for acronyms, i.e. `WSTAudience` / `WSTServerURL`.